Package.swift

```diff
@@ -0,0 +1,51 @@
+// swift-tools-version:5.3
+/*
+ COPYRIGHT 1995-2021 ESRI
+
+ All rights reserved under the copyright laws of the United States
+ and applicable international laws, treaties, and conventions.
+
+ This material is licensed for use under the Esri Master License
+ Agreement (MLA), and is bound by the terms of that agreement.
+ You may redistribute and use this code without modification,
+ provided you adhere to the terms of the MLA and include this
+ copyright notice.
+
+ See use restrictions at http://www.esri.com/legal/pdfs/mla_e204_e300/english
+
+ For additional information, contact:
+ Environmental Systems Research Institute, Inc.
+ Attn: Contracts and Legal Services Department
+ 380 New York Street
+ Redlands, California, USA 92373
+
+ email: contracts@esri.com
+ */
+
+import PackageDescription
+
+let package = Package(
+    name: "ArcGISToolkit",
```

Comment:
Should the name of the package be `arcgis-runtime-toolkit-ios`, to match the name of this repo? Often times that is the case, that the name of the package and the name of the repo containing the package are the same. The library target could still be named `ArcGISToolkit`. Just a thought.

Reply:
Fixed in #148